src/components/SecondaryPanes/Breakpoints/Breakpoints.css

```diff
@@ -183,22 +183,12 @@ html .breakpoints-list .breakpoint.paused {
   inset-inline-end: 15px;
   inset-inline-start: auto;
   position: absolute;
-  top: 8px;
-  display: none;
+  top: -100px;
 }
 
 .breakpoint:hover .close-btn,
-.A11y-keyboard .breakpoint:focus-within .close-btn {
-  display: flex;
-}
-
-.breakpoint .close {
-  visibility: hidden;
-}
-
-.breakpoint:hover .close,
-.A11y-keyboard .breakpoint:focus-within .close {
-  visibility: visible;
+.close-btn:focus {
```

Comment:
This publishes `.close-btn` in the global CSS namespace. Any `<button class="close-btn">` anywhere in Debugger would get those styles, leading to conflicts.

A better selector would be `.breakpoints-list .close-btn:focus`.

Reply:
Thanks a lot for pointing these out! I'm not thoughtful enough.
I'm wondering would adding `.breakpoint` before `.close-btn:focus` be able to keep it consistent with line 182 and 189?

Reply:
Either one should be fine. :)